specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Thanks for digging in to an important topic. It is important for each contest to ensure that the samples we're considering were acquired uniformly and at random. To start out with, we can take the easy approach of only paying attention to a single county and a single contest that is only in it, sample all ballots in the county, ensure that we're looking at all ballots selected in order for that county and ensure we have an answer for each one up to some high water mark, and thus know that our sample is uniformly random.  Some day we will support more efficient sampling of sub-county contests, but we're not there yet.
One really tricky part is that we can't trust the CVRs for knowing exactly which ballots are on which contests. They are what we're auditing and they might not be right. Harvie is working hard on getting independent evidence in the manifest for the ballot style of each ballot, but that's a tall order.